### ISSUE TITLE
sdk4 mediaTracks /  tracks

### ISSUE THREAD

**Issue by chrisdugne** [chaimPaneth/react-native-jw-media-player#189]
Hello, 

From your [docs](https://github.com/chaimPaneth/react-native-jw-media-player#playlistitem) we need to use `playlistitem.tracks`

so I tried:
```
  const playlistItem = {
    [...],
    tracks: [{file:'http://file.com/english.vtt', label:'plop'}]
  };
```

and got (for iOS) `Exception thrown while executing UI block: *** +[JWPlayerKit.JWMediaTrack<0x1109417a0> init]: cannot init a class object.
`

I found then from JWPlayer migration, this field seems to have changed to [mediaTracks](https://developer.jwplayer.com/jwplayer/docs/ios-migration-reference#jwplaylistitem)

So I tried if any chance... and it works...

```
  const playlistItem = {
    [...],
    mediaTracks: [{file:'http://file.com/english.vtt', label:'plop'}]
  };
```

I found one relevant line here https://github.com/chaimPaneth/react-native-jw-media-player/blob/55e468aabb92dba3e2850e4c2f64307e418f3705/ios/RNJWPlayer/RNJWPlayerView.m#L1130

So that's why I write this issue:
- either you should update your doc
- or you should update the code so we can use `tracks`
- or there is something I don't understand and somehow `mediaTracks` is working in my environnment ?!

(and both are working ok for android....!)

thanks for your lib;

**Comment by chaimPaneth** [comment]
Hi @chrisdugne 

I fixed this issue in current latest version 0.2.11

Thank you,
Chaim

**Comment by chrisdugne** [comment]
great, thank you!
